packages/ui/src/composites/wui-account-button/index.ts

```diff
@@ -26,17 +26,19 @@ export class WuiAccountButton extends LitElement {
 
   // -- Render -------------------------------------------- //
   public override render() {
+    const isEns = this.address.endsWith('.eth')
+
     return html`
       <button ?disabled=${this.disabled}>
         ${this.balanceTemplate()}
-        <wui-flex gap="xxs" alignItems="center" class=${this.balance ? undefined : 'noBalance'}>
+        <wui-flex gap="xxs" alignItems="center" class=${!this.balance && 'noBalance'}>
```

Comment:
I think `this.balance ? undefined : 'noBalance'` was more readable to be honest, also let's do `local-no-balance` for class name here, to stick to newly established patterns we have.